### DISCUSSION TITLE
其他人如何使用我的alist

### DISCUSSION THREAD

**Discussion by RenwickQiu** [AlistGo/alist#1930] [Q&A]
是添加我的alist账号吗
可是Alist的地址不都是一样的吗
望解答

**Comment by ihxjie** [comment]
127.0.0.1应该就是你指的同样的alist地址吧，这个是本机回环地址，存在于每个人的电脑上，并不指代某一台特定的主机。如果你是搭建在自己的电脑上，你需要有公网ip地址或通过内网穿透才能分享你的alist，只有这样，别人才能通过你的ip访问到你的电脑，并使用你的alist。如果你是搭建在服务器或者nas上，你应该会有一个公网ip地址，其他人使用这个ip+端口即可访问。
